### PR TITLE
Include only necessary fonts for target environment specified by Browserslist

### LINKED ISSUE #1495: Remove support for TTF fonts
Most browsers including IE 9 support WOFF font format. (https://caniuse.com/#feat=woff) The only concern is `< Android 4.4` but its usage is about 0.2%.

EOT fonts have been removed for similar reasons: https://github.com/Khan/KaTeX/issues/893.

Does anyone have any concerns about removing this font format?

### LINKED ISSUE #1667: Document custom building
- `USE_TTF` option to exclude TTF in added in #1600 
- Browserslist added in #1662 and will be used by #1595

### PR DESCRIPTION
**Pull request by ylemkimon**
Fixes #1667.
Closes #1495.

\+ Allow WOFF and WOFF2 to be controlled using environment variables like TTF. (`USE_TTF`) These environment variables will override target environment.

Uses caniuse data to detect support. No additional dependencies are required, as they are already included in other dependencies.

### CODE REVIEW

File: docs/node.md [hunk 1]
```diff
@@ -36,5 +36,15 @@ yarn
 yarn build
 ```
 
+It will automatically transpile code and include only necessary fonts for
+target environments specified by [Browserslist config](https://github.com/browserslist/browserslist#environment-variables).
+For example, if you are making a web app for a kiosk with Chrome 68, run
+`BROWSERSLIST="Chrome 68" yarn build` and it will produce build with no
```
Comment (kevinbarabash) [inline docs/node.md]: I ran this command got the following error:
```
error "browserslist#node-releases@^1.0.0-alpha.11" doesn't satisfy found match of "node-releases@1.0.0-alpha.10"
error "browserslist#node-releases" is wrong version: expected "1.0.0-alpha.11", got "1.0.0-alpha.10"
```

Comment (ylemkimon) [reply docs/node.md]: @kevinbarabash This PR updates some package versions by `yarn.lock`. You need to `yarn install` first. (If you did, maybe due to recent NPM downtime: https://status.npmjs.org/)

Comment (kevinbarabash) [reply docs/node.md]: If there a difference between running `yarn` and `yarn install`?  It looks like the `circleci: test` check is running into this issue too.

Comment (ylemkimon) [reply docs/node.md]: @kevinbarabash AFAIK, `yarn` and `yarn install` is equivalent: https://yarnpkg.com/en/docs/cli/#toc-default-command. The CircleCI build failures are due to old Yarn version (which #1658 addresses).

**Comment by ylemkimon** [comment]
Test failures are due to the same issue in #1648, which'll be fixed by #1658.

File: docs/node.md (hunk 1 above)
Comment (ylemkimon) [reply docs/node.md]: @kevinbarabash The error seems to be fixed!

**Comment by ylemkimon** [comment]
@kevinbarabash There was a bug in #1595, where Browserslist config is not read even if `isESMBuild` is false. Also, I've upgraded `uglifyjs-webpack-plugin` to `terser-webpack-plugin`, to be able to minify ES6 output.

**Pull request merged**